File: kotlin/v3/src/main/kotlin/com/api/v3/persons/PersonResponseMapper.kt
package com.api.v3.persons

import com.api.v3.persons.domain.Person
import com.api.v3.persons.dtos.PersonResponseDto

class PersonResponseMapper {

    companion object {
        fun map(person: Person): PersonResponseDto {
            return PersonResponseDto(
                person.fullName(),
                person.ssn,
                person.birthDate,
                person.email,
                person.address,
                person.gender,
                person.phoneNumber
            )
        }
    }

}

File: c#/v4/People/Utils/PersonResponseMapper.cs
using v4.People.Domain;
using v4.People.DTOs;

namespace v4.People.Utils;

public class PersonResponseMapper
{
    public static PersonResponseDto Map(Person person)
    {
        return new PersonResponseDto(
            person.FullName(),
            person.Ssn,
            person.BirthDate,
            person.Email,
            person.PhoneNumber,
            person.Address
        );
    }
}